site.json

```diff
@@ -2,7 +2,7 @@
     "bundles": {
         "codeMirror": {
             "javascript": ["js/libs/codemirror.js", "js/libs/javascript.js"],
-            "css": ["css/libs/codemirror.css", "css/editor-libs/codemirror-override.css"],
+            "css": ["css/libs/codemirror.css"],
```

Comment:
Whoops, I believe this is "your master was behind mdn master when you created the feature branch" but, we do not want to remove that second stylesheet.

Reply:
Good spot. I'll add it back in.